weave

```diff
@@ -620,6 +620,10 @@ try_create_bridge() {
             # across our bridge. E.g. ufw
             add_iptables_rule filter FORWARD -i $BRIDGE -o $BRIDGE -j ACCEPT
         fi
+        # Forward from weave to the rest of the world
+        add_iptables_rule filter FORWARD -i $BRIDGE ! -o $BRIDGE -j ACCEPT
+        # and allow replies back
+        add_iptables_rule filter FORWARD -o weave -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT
```

Comment:
I am wondering why this rule has `-o weave` and not `-o $BRIDGE`? (given we set: `BRIDGE=weave` earlier)
Is it guaranteed that there will always be a bridge named `weave`? (it seems that `create_bridge` in the `weave` script always uses `$BRIDGE`, so I'm wondering if this could break if someone sets `BRIDGE` to a different value than the default)